apps/src/code-studio/components/AlertExclamation.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * A big blue box with an exclamation mark on the left and text on the right.
+ */
+export default function AlertExclamation({children}) {
+  const cyan = '#0094ca';
+  const style = {
```

Comment:
Is there a reason all of these styles are defined inside the component, rather than using the `styles` pattern?

Reply:
🤷‍♀️ I also don't know why we're using a custom color here

I'll move these styles down